### PR TITLE
Make the plain-old smileys useful. Closes #697

### LINKED ISSUE #697: Please bring be the simple emjois
As I'm using a smartphone with android 4.2 installed (not updatable) , I can't use emjois anymore.
So please bring back a alternative way to use the old, simple ones :-)

### PR DESCRIPTION
**Pull request by marmistrz**

### CODE REVIEW

File: app/src/main/res/xml/popup_smileys.xml [hunk 1]
```diff
@@ -4,27 +4,27 @@
           android:keyWidth="15%p"
           android:keyHeight="@integer/key_normal_height">
     <Row android:rowEdgeFlags="top">
-        <Key android:keyLabel=":-(" android:keyOutputText=":-(\u0020" android:keyEdgeFlags="left"/>
-        <Key android:keyLabel=":_(" android:keyOutputText=":_(\u0020"/>
-        <Key android:keyLabel="8-[" android:keyOutputText="8-[\u0020"/>
-        <Key android:keyLabel="xx-P" android:keyOutputText="xx-P\u0020"/>
-        <Key android:keyLabel="|-." android:keyOutputText="|-.\u0020"/>
-        <Key android:keyLabel="}-]" android:keyOutputText="}-]\u0020" android:keyEdgeFlags="right"/>
+        <Key android:keyLabel=":)" android:keyOutputText=":)\u0020" android:keyEdgeFlags="left"/>
```
Comment (menny) [inline app/src/main/res/xml/popup_smileys.xml]: I see that `:)` is missing, thanks for adding it.

File: app/src/main/res/xml/popup_smileys.xml [hunk 2]
```diff
@@ -4,27 +4,27 @@
           android:keyWidth="15%p"
           android:keyHeight="@integer/key_normal_height">
     <Row android:rowEdgeFlags="top">
-        <Key android:keyLabel=":-(" android:keyOutputText=":-(\u0020" android:keyEdgeFlags="left"/>
-        <Key android:keyLabel=":_(" android:keyOutputText=":_(\u0020"/>
-        <Key android:keyLabel="8-[" android:keyOutputText="8-[\u0020"/>
-        <Key android:keyLabel="xx-P" android:keyOutputText="xx-P\u0020"/>
-        <Key android:keyLabel="|-." android:keyOutputText="|-.\u0020"/>
-        <Key android:keyLabel="}-]" android:keyOutputText="}-]\u0020" android:keyEdgeFlags="right"/>
```
Comment (menny) [inline app/src/main/res/xml/popup_smileys.xml]: `}-]` is quite popular. Please add it back.

File: app/src/main/res/xml/popup_smileys.xml [hunk 3]
```diff
@@ -4,27 +4,27 @@
...
-        <Key android:keyLabel=":-I" android:keyOutputText=":-I\u0020"/>
-        <Key android:keyLabel="8-." android:keyOutputText="8-.\u0020"/>
-        <Key android:keyLabel="8-o" android:keyOutputText="8-o\u0020"/>
-        <Key android:keyLabel="\\m/" android:keyOutputText="\\m/\u0020"/>
-        <Key android:keyLabel="O_Q" android:keyOutputText="O_Q\u0020" android:keyEdgeFlags="right"/>
+        <Key android:keyLabel=":/" android:keyOutputText=":/\u0020" android:keyEdgeFlags="left"/>
+        <Key android:keyLabel=":S" android:keyOutputText=":S\u0020"/>
+        <Key android:keyLabel=":\'(" android:keyOutputText=":\'(\u0020"/>
+        <Key android:keyLabel=":\')" android:keyOutputText=":\')\u0020"/>
+        <Key android:keyLabel="\o/" android:keyOutputText="\o/\u0020"/>
```
Comment (menny) [inline app/src/main/res/xml/popup_smileys.xml]: You need to escape the `\`. Please do: `<Key android:keyLabel="\\o/" android:keyOutputText="\\o/\u0020"/>`

File: app/src/main/res/xml/popup_smileys.xml [hunk 4]
```diff
@@ -4,27 +4,27 @@
...
+        <Key android:keyLabel=":P" android:keyOutputText=":P\u0020"/>
+        <Key android:keyLabel=":*" android:keyOutputText=":*\u0020" android:keyEdgeFlags="right"/>
     </Row>
     <Row>
-        <Key android:keyLabel=":-X" android:keyOutputText=":-X\u0020" android:keyEdgeFlags="left"/>
-        <Key android:keyLabel=":-I" android:keyOutputText=":-I\u0020"/>
-        <Key android:keyLabel="8-." android:keyOutputText="8-.\u0020"/>
-        <Key android:keyLabel="8-o" android:keyOutputText="8-o\u0020"/>
-        <Key android:keyLabel="\\m/" android:keyOutputText="\\m/\u0020"/>
-        <Key android:keyLabel="O_Q" android:keyOutputText="O_Q\u0020" android:keyEdgeFlags="right"/>
```
Comment (menny) [inline app/src/main/res/xml/popup_smileys.xml]: `O_Q` is also very popular, please add that back

Comment (marmistrz) [inline app/src/main/res/xml/popup_smileys.xml]: What does that mean btw.? Seems very similar to `o_o`

File: app/src/main/res/xml/popup_smileys.xml (hunk 2 above)
Comment (marmistrz) [inline app/src/main/res/xml/popup_smileys.xml]: What does that mean btw?

File: app/src/main/res/xml/popup_smileys.xml (hunk 4 above)
Comment (menny) [inline app/src/main/res/xml/popup_smileys.xml]: it's the same thing, but with a monocle.

**Comment by marmistrz** [comment]
So I'd add `O_Q`, `}-]`, `xD`. I'd simply create a new row: do you remember two other common ones?

**Comment by menny** [comment]
not really, but you can also let ASK figure row sizes for you.
In other quick-text plugins, there is only one row, and ASK split it when it needs to.
See https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/master/app/src/main/res/xml-v22/quick_text_unicode_food.xml

**Comment by marmistrz** [comment]
I'll fix that soon.

**Comment by marmistrz** [comment]
@menny  is it ok now?

**Comment by menny** [comment]
ya, looks good.

**Comment by menny** [comment]
Should I merge?

**Comment by marmistrz** [comment]
Sure

**Comment by menny** [comment]
I'll release a new beta in the coming days.

**Pull request merged**